.gitignore

```diff
@@ -22,9 +23,9 @@ tags
 /docs/build
 /ployst.egg-info
 /static
+/.tox/
 
 # Some OSX specific thing
 *.DS_Store
-/.tox/
 
 ployst/github/test/data/
```

Comment:
Maybe we need the generated files to live in a different root folder altogether? I don't like mixing up folders with VC'd and non-VC'd stuff